acceptance/tests/ensure_version_file.rb

```diff
@@ -6,7 +6,7 @@
 
 test_name 'PA-466: Ensure version file is created on agent' do
 
-  skip_test 'requires version file which is created by AIO' if [:gem, :git].include?(@options[:type])
+  skip_test 'requires version file which is created by AIO' if @options[:type] != 'aio'
```

Comment:
i'd really like to change the type name here to 'package' or something similar.  but it's fine to now do that in this PR.